.vitepress/sidebar.ts

```diff
@@ -175,6 +185,48 @@ export function getSidebar() {
             text: 'Debugging',
             link: '/guides/build-iapp/debugging',
           },
+          {
+            text: 'Advanced Low‑Level iApp Building',
```

Comment:
```suggestion
            text: 'Advanced',
```
To be consistent with the tools terminology, only use 'Advanced'.
I think this is the goal of overview page to explain this Advanced section. No need to say more in the title.

Reply:
fix Commit: 4ca729f299b9843ac4eceacaf9e63cf55c1eb236